anthos-bm-apigee/README.md

```diff
@@ -0,0 +1,81 @@
+    - The Service Account has `Owner` and `Apigee Organization Administrator` permissions
+    - The Service Account has both `Editor`, `Project IAM Admin` and `Apigee Organization Administrator` permissions
+The scripts to create service account and key creation is also mentioned in Quickstart module.
+- Organizational Policy Constraints
+      Follwowing list of Policy needs to be enabled for the organizations. If these are not enabled at Organizations, please consider them for the project.
+
+      |  Policy Name                                 | Constraint Name                                   | Effective Polciy |
+      |  ------------------------------------------- | ------------------------------------------------- | ---------------- |
+      | Disable service account creation             | constraints/iam.disableServiceAccountCreation	   | Not Enforced     |
+      | Disable service account key creation         | constraints/iam.disableServiceAccountKeyCreation	 | Not enforced     |
+      | Restrict VM IP Forwarding                    | constraints/compute.vmCanIpForward	         | Allowed All      |
+      | Define allowed external IPs for VM instances | constraints/compute.vmExternalIpAccess	           | Allowed All      |
+      | Shielded VMs                                 | constraints/compute.requireShieldedVm	           | Not Enforced     |
+      | Require OS Login                             | constraints/compute.requireOsLogin	               | Not Enforced     |
+      | Skip default network creation                | constraints/compute.skipDefaultNetworkCreation	   | Not Enforced     |
+
+- Default Network with default Firewall policies
+The installation requires a network with name as default. If default network creation is enabled for the organization, the project will get them inherited. In case the  Skip default network creation is Enforced, you can create a new VPC network with name default in auto mode.
+![Default Network](docs/images/default_network.png)
+
+- Quota Check
+The demo Apigee instance requires 4 VMs with n1-standard-8 machine type. Please ensure there are enough quota set for CPU,Memory, IP Addresses for the region you are hosting the project.
+
+- **Sample Prerequisite script is also provided unnder resources folder as ./resources/run_prerequisite.sh**
+
+### Bare metal infrastructure on Google Cloud using Compute Engine VMs
+
+The [Quick starter](docs/quickstart.md) guide sets up the following infrastructure in Google Cloud using Compute Engine VMs. The diagram assumes that the none of the default values for the [variables](variables.tf) were changed other than the ones mentioned in the quick starter.
+
+![Bare metal infrastructure on Google Cloud using Compute Engine VMs](docs/images/abm_gcp_infra.png)
+
+---
+## Getting started
+
+- [Terraform Module Information _(includes variables definitions)_](docs/variables.md)
```

Comment:
This link is broken. I will generate it before merging my PR. Please do remind me on my PR if in case I forget

Reply:
Yes sure